lib/ansible/plugins/connection/network_cli.py

```diff
@@ -757,3 +788,10 @@ def _get_terminal_std_re(self, option):
             terminal_std_re = getattr(self._terminal, option)
 
         return terminal_std_re
+
+    def _receive_data(self, buffer):
+        if self._transport_type == 'libssh':
+            while True:
```

Comment:
Did you mean?
```suggestion
            while not self._ssh_shell.poll():
                pass
```
Also, maybe it's worth adding some small sleep?